BUILD.boost

```diff
@@ -82,6 +82,19 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+# TODO(https://github.com/bazelbuild/bazel-skylib/issues/191): use
```

Comment:
For future reference, I think the bazel bug here is https://github.com/bazelbuild/bazel/issues/4856